packages/backend-function/runtime/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@aws-amplify/backend-function/runtime",
+  "main": "../dist/cjs/runtime/index.js",
+  "react-native": "../src/runtime/index.ts",
+  "browser": "../dist/esm/runtime/index.mjs",
+  "module": "../dist/esm/runtime/index.mjs",
+  "typings": "../dist/esm/runtime/index.d.ts"
+}
```

Comment:
Why do we need this file ?

Reply:
I'm not sure. I thought I had errors without it, but can't replicate them this morning. This is the pattern that is followed in js and elsewhere, but I'll remove it until/unless I encounter the issue it resolves.